NAMESPACE

```diff
@@ -2,29 +2,27 @@
 
 export(check_zero_tally)
 export(gtable_filter_remove)
-export(launch_shiny_report)
 export(match_concepts)
 export(mini_dict)
 export(plot_age)
 export(plot_ethnicity)
 export(plot_sex)
-export(plot_tm)
 export(plot_visit_profile)
 export(prepare_cohort_boundaries)
 export(prepare_null_counts)
 export(prepare_overview)
 export(prepare_tables)
 export(prepare_tally)
-export(render_report)
+export(render_quality_report)
+export(render_summary_report)
 export(summarise_missing)
 export(tally_concept)
 export(theme_d)
```

Comment:
Can you check that all these exported functions are actually being used? 
If not, please delete them.

Reply:
I removed the unnecessary functions. Please check now